packages/cli/src/profiles.ts

```diff
@@ -0,0 +1,59 @@
+import { Command } from 'commander';
+import { createMedplumCommand } from './util/command';
+import { FileSystemStorage } from './storage';
+import { resolve } from 'path';
+import { readdirSync } from 'fs';
+import { homedir } from 'os';
+
+const setProfile = createMedplumCommand('set');
+const removeProfile = createMedplumCommand('remove');
+const listProfiles = createMedplumCommand('list');
+const describeProfile = createMedplumCommand('describe');
+
+export const profile = new Command('profile')
+  .addCommand(setProfile)
+  .addCommand(removeProfile)
+  .addCommand(listProfiles)
+  .addCommand(describeProfile);
+
+setProfile
+  .argument('<profileName>', 'Name of the profile')
+  .description('Create a new profile or replace it with the given name and its associated properties')
+  .action(async (profileName, options) => {
+    const storage = new FileSystemStorage(profileName);
+    storage.setObject('profile', options);
```

Comment:
Thoughts on whether this property should be called "profile" vs "options" vs "config" vs ???

Reply:
setting to "options"